README.md

```diff
@@ -1 +1,29 @@
-"# project24" 
+
+##internet-shop
+Internet-shop with basic functions. 
+According to N-tier architecture project contain several layers:
+* DB
+* DAO
+* Services
+* Controllers
+* Views
+##main functions
+###for Admin: 
+* Manage products
+* Manage users
+* View and delete orders
+###for User:
+* Registration
+* Login/Logout
+* Buy products
+* View Shopping Cart
+* Create orders
+* View own orders and its details
+##technologies 
+Java, Servlets, RBAC Filters, Tomcat, JSP, JSTL, JDBC, MySQL
```

Comment:
I think you could add the version of Java you used (e.g. Java 8, Java 11) and format this list as bullet points, e.g.
- Java 8
- Servlets

Reply:
Done